### ISSUE TITLE
Issues between builds 0703597 to a0c73ed

### ISSUE THREAD

**Issue by KitBallard64** [dimok789/loadiine_gx2#96]
Figured I'll keep track of what I've been having an issue with here, since they get buried on the forum. Starting at build e71fb52, Minecraft no longer loads via Smash boot, ending in an "SD Read" Error (Error 150-3032, if I'm remembering right), which leads me to go HOME. Last working version was 0703597, as it loads the game fine. Been trying to wonder if it's just me or not. Tried it with Same Save setting and gives the same error.

Boot Settings for Minecraft:
No Update/DLC
Smash Bros Boot
Unique Save

Also, noticed an oddity in a0c73ed, where turning on HID pad setting while having a Ps4 Remote plugged in will freeze the boot.  Also had the GC USB adapter plugged in but it launches fine it plugged in. Version 0703597 worked normally again.

Hoping that I'm not the only one having these oddities.

**Comment by jaxrogers2** [comment]
Yeah, Minecraft v0 is no longer working on newest build, haven't tried using the update yet though.

**Comment by KitBallard64** [comment]
It'll do the same thing. I'm on the newest (v64).
